src/managers/GuildBanManager.js

```diff
@@ -142,15 +148,30 @@ class GuildBanManager extends CachedManager {
    *   .then(banInfo => console.log(`Banned ${banInfo.user?.tag ?? banInfo.tag ?? banInfo}`))
    *   .catch(console.error);
    */
-  async create(user, options = { days: 0 }) {
+  async create(user, options = {}) {
     if (typeof options !== 'object') throw new TypeError('INVALID_TYPE', 'options', 'object', true);
     const id = this.client.users.resolveId(user);
     if (!id) throw new Error('BAN_RESOLVE_ID', true);
+
+    if (typeof options.days !== 'undefined' && !deprecationEmittedForDays) {
+      process.emitWarning(
+        'The days option for GuildBanManager#create() is deprecated. Use the deleteMessageSeconds option instead.',
+        'DeprecationWarning',
+      );
+
+      deprecationEmittedForDays = true;
+    }
+
     await this.client.api
       .guilds(this.guild.id)
       .bans(id)
       .put({
-        data: { delete_message_days: options.days },
+        data: {
+          delete_message_seconds:
```

Comment:
Same requested change as in the main PR (is it optional, if it is we shouldn't enforce it)

Reply:
This would be a breaking change since it is a default.

Reply:
[Internally resolved.](https://canary.discord.com/channels/222078108977594368/999796393810067607/1014275925812183150)